swagger/swagger.go

```diff
@@ -44,8 +44,19 @@ func ImportStatements(imports []string) string {
 	if len(imports) == 0 {
 		return ""
 	}
+	remoteImports := []string{}
```

Comment:
ah sorry, on a whim I changed this so that imports follow `goimports` style of putting built-in imports at the top and third party imports at the bottom